<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenhuaiNew</class>
 <widget class="QWidget" name="RenhuaiNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="title_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>103</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>-1</y>
      <width>421</width>
      <height>103</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>15</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>当前登录：</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentUserLabel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>15</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutPushButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>70</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>登入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="switchUserPushButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>70</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>切换</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>30</y>
      <width>850</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);</string>
    </property>
    <property name="text">
     <string>污泥好氧发酵智能控制系统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="left_banner_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>138</y>
     <width>172</width>
     <height>855</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>55</y>
      <width>152</width>
      <height>752</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="loginButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/image/new/login_unselected.bmp</normalon>
         <activeon>:/image/new/login_selected.bmp</activeon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ferControlButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/image/new/fercontrol_unselected.bmp</normalon>
         <activeon>:/image/new/fercontrol_selected.bmp</activeon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="fanControlButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/image/new/fancontrol_unselected.bmp</normalon>
         <activeon>:/image/new/fancontrol_selected.bmp</activeon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="fanValveControlButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/image/new/sensorcontrol_unselected.bmp</normalon>
         <activeon>:/image/new/sensorcontrol_selected.bmp</activeon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="SettingButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/image/new/setting_unselected.bmp</normalon>
         <activeon>:/image/new/setting_selected.bmp</activeon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="HistoryQueryButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/image/new/historychart_unselected.bmp</normalon>
         <activeon>:/image/new/historychart_selected.bmp</activeon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="realTimeChartButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/image/new/realtimechart_unselected.bmp</normalon>
         <activeon>:/image/new/realtimechart_selected.bmp</activeon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ReportButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/image/new/report_query_unselected.bmp</normalon>
         <activeon>:/image/new/report_query_selected.bmp</activeon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="AlertQueryButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/image/new/alert_unselected.bmp</normalon>
         <activeon>:/image/new/alert_selected.bmp</activeon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="exitButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/image/new/exit_unselected.bmp</normalon>
         <activeon>:/image/new/exit_selected.bmp</activeon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="tab_banner_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>104</y>
     <width>1920</width>
     <height>33</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/image/new/tab_banner.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>40</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>-15</x>
     <y>985</y>
     <width>1950</width>
     <height>109</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/new/buttom_banner.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>25</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);
background: transparent;</string>
    </property>
    <property name="text">
     <string>版权所有 北京合清环保技术有限公司</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>55</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);
background: transparent;</string>
    </property>
    <property name="text">
     <string>技术服务 010-84375388</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>33</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);
background: transparent;</string>
    </property>
    <property name="text">
     <string>当前连接主机：</string>
    </property>
   </widget>
   <widget class="QLabel" name="hostname_label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>33</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);
background: transparent;</string>
    </property>
    <property name="text">
     <string>106</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="serverIP_label">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>33</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);
background: transparent;</string>
    </property>
    <property name="text">
     <string>106</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>33</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);
background: transparent;</string>
    </property>
    <property name="text">
     <string>当前连接主机：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>33</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);
background: transparent;</string>
    </property>
    <property name="text">
     <string>系统状态：</string>
    </property>
   </widget>
   <widget class="QLabel" name="systemstatus_label">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>33</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);
background: transparent;</string>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="uploadtime_label">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>60</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);
background: transparent;</string>
    </property>
    <property name="text">
     <string>2018-01-01 00：00：00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);
background: transparent;</string>
    </property>
    <property name="text">
     <string>系统上次上传时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>25</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);
background: transparent;</string>
    </property>
    <property name="text">
     <string>17:17:17</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_label">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>55</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);
background: transparent;</string>
    </property>
    <property name="text">
     <string>2018-01-01</string>
    </property>
   </widget>
   <widget class="QLabel" name="week_label">
    <property name="geometry">
     <rect>
      <x>1840</x>
      <y>55</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 250, 250);
background: transparent;</string>
    </property>
    <property name="text">
     <string>星期一</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="main_content_frame">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>102</y>
     <width>1731</width>
     <height>891</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1731</width>
      <height>891</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="main_content_frame_horizontalLayout"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
